docs/01-Chart-Configuration.md

```diff
@@ -257,6 +257,18 @@ afterBody | `Array[tooltipItem], data` | Text to render after the body section
 beforeFooter | `Array[tooltipItem], data` | Text to render before the footer section
 footer | `Array[tooltipItem], data` | Text to render as the footer
 afterFooter | `Array[tooltipItem], data` | Text to render after the footer section
+dataPoints | `Array[CustomTooltipDataPoint], data` | List of matching point informations.
+
+#### CustomTooltipDataPoint
+
+Name | Type | Description
+--- | --- | ---
+index | Number | Matching point index.
+datasetIndex | Number | Matching dataset index.
+xLabel | String | Matching label on X axis.
+yLabel | String | Matching label on Y axis.
+pointX | Number | X position of matching point.
```

Comment:
Just thinking, while the containing object is`dataPoints`, wouldn't `x` and `y` be better names?
```javascript
dataPoints.forEach(function(point) {
    point.x; // compared to point.pointX
    point.y; // compared to point.pointY
});
```
Would also be more consistent with `index` (and not `pointIndex`).